README.md

```diff
@@ -10,27 +10,20 @@ An operator to perform lifecycle management (install/upgrade/uninstall) of [Kata
 
 1. Make sure that `oc` is configured to talk to the cluster
 
-2. Clone the Kata Operator repository and check out the branch matching with the Openshift version. e.g. If you are running Openshift 4.7 then,
+2. To deploy the operator and create a custom ressource (which installs Kata on all worker nodes), run
+   ``` curl https://raw.githubusercontent.com/openshift/kata-operator/master/deploy/install.sh | bash ```
 
+  This will create all necessary ressources, deploy the kata-operator and also create a custom ressource.
```

Comment:
s/ressource/resource

Reply:
ah, it's spelled with ss in german :-) thx